pkgs/by-name/hi/hiddify-next/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  appimageTools,
+  fetchurl,
+}:
+let
+  pname = "hiddify-next";
+  version = "1.1.1";
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://github.com/hiddify/hiddify-next/releases/download/v${version}/Hiddify-Linux-x64.AppImage";
+    hash = "sha256-T4BWxhJ7q13KE1rvvFsnXhs2XVEmNkFTJbJ4e8PCg+0=";
+  };
+  hiddify = appimageTools.wrapType2 {
+    inherit pname version src;
+
+    extraPkgs = pkgs: with pkgs; [ libepoxy ];
+  };
+in
+stdenv.mkDerivation rec {
```

Comment:
Not needed to open another derivation. See example: https://github.com/NixOS/nixpkgs/blob/efcd11b433edce0b444180e23bcd66f964050dd9/pkgs/by-name/ja/jan/package.nix